>From 175786a0008092225644aa684e4b8a5bfa921015 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sun, 4 Jan 2015 12:49:00 +0900
Subject: [PATCH 1/2] add emacsen-compat for modern Emacs lisp package

as debian-emacs-policy, newer style emacs package should add compat
file, see /usr/share/doc/emacsen-common/debian-emacs-policy.gz

5) Emacs add-on package support (examples below should make this much clearer)
>
>  A) Each add-on package must add a "Depends: emacsen-common (>=
>     2.0.8)" and include a file like this that indicates its
>     emacsen-common compatibility level:
>
>       /usr/lib/emacsen-common/packages/compat/<package>
>
>     This file should contain a single integer, which at the moment
>     should be 0.  (Currently, the file is just used to distinguish
>     between old-style packages and new-style packages.)
---
 lib/emacs/emacsen-compat | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 lib/emacs/emacsen-compat

diff --git a/lib/emacs/emacsen-compat b/lib/emacs/emacsen-compat
new file mode 100644
index 0000000..573541a
--- /dev/null
+++ b/lib/emacs/emacsen-compat
@@ -0,0 +1 @@
+0
-- 
2.1.4

